README.md

```diff
@@ -15,7 +15,7 @@ Boom! Turns out you probably can't delete that intern's CSS from 6 months ago th
 Install from npm
 
 ```bash
-npm install cypress-image-snapshot
+npm install -D cypress-image-snapshot
```

Comment:
I think it might be clearer if we use the full `--save-dev` flag as not everyone is familiar with the shorthand.

Reply:
Ha, interesting. For the exact same reason I tend to keep the shorthand -- just in case someone didn't know about it. :smile: Changed. (You can squash the commits via GitHub interface, right? Or will I have to rebase on my own? I'm being lazy an trying to avoid cloning locally :sweat_smile: )